pkg/metaserver/constants.go

```diff
@@ -59,6 +59,11 @@ type PatchInfo struct {
 	Subresources []string
 }
 
+type RestartInfo struct {
```

Comment:
Defining structures in the constants.go is not good, you can add a file types.go to maintain them, the struct `RestartResponse` can also be moved over to this file, it is not good that the keadm module depend on the metaserver module package.

Reply:
Done